functions/Start-DbaXESession.ps1

```diff
@@ -86,11 +86,15 @@ function Start-DbaXESession {
 				$session = $xe.Name
 				if (-Not $xe.isRunning) {
 					Write-Message -Level Verbose -Message "Starting XEvent Session $session on $instance."
-					$xe.Start()
-					Get-DbaXESession -SqlInstance $xe.Parent -Session $session
+					try {
+						$xe.Start()
+					} catch {
+						Write-Error -Message "Could not start XEvent Session $session on $instance"
```

Comment:
This should use `Stop-Function` this is part of the message system in the module. Supports not showing the sea of read all the time.

So in this case you would have something like `Stop-Function -Message "Could not start XEvent Session on $instance" -Target $session -ErrorRecord $_`

Reply:
OK.  I added a commit with the changes.